recipes/opensubdiv/all/conanfile.py

```diff
@@ -33,14 +33,14 @@ class OpenSubdivConan(ConanFile):
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_tbb": False,
-        "with_opengl": False,
-        "with_omp": False,
+        "with_tbb": True,
```

Comment:
I need a bit more insight into this. Are there the default values in upstream? Or is it just that these values are more convinient to any user for the recipe, and specifically your releated PR? Thanks!

Reply:
Hi @AbrilRBS , in https://github.com/PixarAnimationStudios/OpenSubdiv/blob/release/CMakeLists.txt#L323 we can see that by default all options are enabled with a double negation.

Reply:
This is a prerequisite of https://github.com/conan-io/conan-center-index/pull/24506 on which there is yet many works, notably plugins managment in package_info() for which I have send you a private message (https://cpplang.slack.com/archives/D071ARHMDL0/p1727905406255289).